UI/Source/CollectionViews/mac/CollectionViewController.swift

```diff
@@ -270,4 +276,10 @@ open class CollectionViewController: ModelCollectionViewController, CollectionVi
     private func copy(_ sender: Any) {
         selectedViewModel()?.handleUserEvent(.copy)
     }
+
+    @objc
+    private func doubleClick(_ sender: Any) {
```

Comment:
We'll probably eventually want a function thats `open` where subclasses can override this behavior but we can avoid exposing that till we have a use case (also not sure whether it should expose the event based on an indexPath like `collectionView(collectionView:didClickIndexPath:)` which would be weird for outline view, or if it should be `didDoubleClickModel:`)